### PR TITLE
Add growable popcorn and horseradish

### PR DESCRIPTION
**Pull request by Rail-Runner**
#### Summary

```SUMMARY: Content "Add growable popcorn and horseradish"```
#### Purpose of change

Expand choices of food which can be created post-Cataclysm.
#### Describe the solution

Add growable popcorn and horseradish plants, as well as some recipes involving items obtained from those plants: horseradish condiment, popcorn kernels, hominy and buttered popcorn. Also added newly-added items to already existing recipes.
#### Describe alternatives you've considered

Considered adding tea as well, but couldn't find information on anyone growing it outdoors successfully in New England.
#### Testing

Copy-pasted changes to game files and verified that no errors happen, looked at amount of calories in crafting screen to make sure that crafted and used amounts of items make sense.
#### Additional context

None.

### CODE REVIEW

File: data/json/items/comestibles/raw_veggy.json [hunk 1]
```diff
@@ -524,6 +561,26 @@
...
+    "type": "COMESTIBLE",
+    "id": "horseradish_greens",
+    "name": { "str": "horseradish greens", "str_pl": "horseradish greens" },
+    "weight": "50 g",
+    "color": "light_green",
+    "spoils_in": "6 days 16 hours",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 14,
```
Comment (LaVeyanFiend) [inline data/json/items/comestibles/raw_veggy.json:574]:
```suggestion
    "calories": 16,
```

File: data/json/items/comestibles/raw_veggy.json [hunk 2]
```diff
@@ -524,6 +561,26 @@
...
+    "description": "Leaves of a horseradish plant with a sharp, bitter and peppery taste.  They're edible, but not commonly eaten.",
+    "price": 80,
+    "material": "veggy",
+    "volume": "100 ml",
+    "fun": -2,
+    "flags": [ "FREEZERBURN", "SMOKABLE", "RAW" ],
+    "smoking_result": "dry_veggy",
+    "vitamins": [ [ "vitA", 136 ], [ "vitC", 34 ], [ "calcium", 4 ], [ "iron", 2 ] ]
```
Comment (LaVeyanFiend) [inline data/json/items/comestibles/raw_veggy.json:582]:
```suggestion
    "vitamins": [ [ "vitA", 7 ], [ "vitC", 30 ], [ "calcium", 2 ], [ "iron", 3 ] ]
```

**Comment by Zireael07** [comment]
"Growable popcorn plant"? How does that look like?

**Comment by anothersimulacrum** [comment]
It's a corn plant, but it's not as sweet as normal corn (sweetcorn), and has some other differences, IIRC.

**Comment by Zireael07** [comment]
Aaa, thanks - the wording made me think of, err, literally growing popcorn itself (GMO of the future xDDD)

**Pull request merged**